GetVisibleServices/handler.ts

```diff
@@ -52,7 +53,7 @@ export function GetVisibleServicesHandler(
       maybeVisibleServicesJson => {
         const servicesTuples = toServicesTuple(
           new StrMap(maybeVisibleServicesJson.getOrElse({}))
-        );
+        ).filter(_ => _.scope === ServiceScopeEnum.NATIONAL); // Temporary disable Local services
```

Comment:
We can add an app setting with the list of enabled services by scope, for example
`FF_SERVICE_SCOPES="LOCAL,NATIONAL"`

Reply:
Added a FF to enable NATIONAL scope filtering in 10f69cf. IMHO is useless a deep filtering configuration with FF.

Reply:
new app setting will be `FF_ONLY_NATIONAL_SERVICES` with true/false values?

Reply:
Yes. The variable is optional if missing the filter is disabled.

Reply:
ok I will add this app setting on fn-apps and we can deploy only to one fn-app to test it

Reply:
FYI https://github.com/pagopa/io-infrastructure-live-new/pull/421